### ISSUE TITLE
Torsocks freezes on android 8.1 aarch64

### ISSUE THREAD

**Issue by Chubukov-Aleksey** [termux/termux-packages#2711]
Torsocks stucks before executing any torified command. 
 
```
Updatable packages:
All packages up to date
System information:
Linux localhost 3.18.71-perf-gebb0eb7 #1 SMP PREEMPT Thu Jul 12 20:45:23 WIB 2018 aarch64 Android
Termux-packages arch:
aarch64
Android version:
8.1.0
Device manufacturer:
Xiaomi
Device model:
Mi A1
```

  
 strace log: https://gist.githubusercontent.com/Chubukov-Aleksey/27bd6b9718e2394562977560801904ae/raw/2499ed88760d76f8f9d3417ad8328edc4fef71eb/strace%20torsocks%20ls.txt

**Comment by Grimler91** [comment]
No error/warning whatsoever?

**Comment by Chubukov-Aleksey** [comment]
No output from torsocks or proxified command. And also no luck with resizing terminal: coomand catches SIGWINCH an then returns to futex:
```
openat(AT_FDCWD, "/data/data/com.termux/files/usr/etc/tor/torsocks.conf", O_RDONLY) = 4                                                                                                         futex(0x7a70e9aa98, FUTEX_WAIT_BITSET_PRIVATE, 2, NULL, 0xffffffff) = ? ERESTARTSYS (To be restarted if SA_RESTART is set)
--- SIGWINCH {si_signo=SIGWINCH, si_code=SI_KERNEL} ---
futex(0x7a70e9aa98, FUTEX_WAIT_BITSET_PRIVATE, 2, NULL, 0xffffffff) = ? ERESTARTSYS (To be restarted if SA_RESTART is set)
--- SIGWINCH {si_signo=SIGWINCH, si_code=SI_KERNEL} ---
futex(0x7a70e9aa98, FUTEX_WAIT_BITSET_PRIVATE, 2, NULL, 0xffffffff) = ? ERESTARTSYS (To be restarted if SA_RESTART is set)
--- SIGWINCH {si_signo=SIGWINCH, si_code=SI_KERNEL} ---
futex(0x7a70e9aa98, FUTEX_WAIT_BITSET_PRIVATE, 2, NULL, 0xffffffff) = ? ERESTARTSYS (To be restarted if SA_RESTART is set)
--- SIGWINCH {si_signo=SIGWINCH, si_code=SI_KERNEL} ---
futex(0x7a70e9aa98, FUTEX_WAIT_BITSET_PRIVATE, 2, NULL, 0xffffffff) = ? ERESTARTSYS (To be restarted if SA_RESTART is set)
--- SIGWINCH {si_signo=SIGWINCH, si_code=SI_KERNEL} ---
futex(0x7a70e9aa98, FUTEX_WAIT_BITSET_PRIVATE, 2, NULL, 0xffffffff) = ? ERESTARTSYS (To be restarted if SA_RESTART is set)
--- SIGWINCH {si_signo=SIGWINCH, si_code=SI_KERNEL} ---                                                                                                                                         futex(0x7a70e9aa98, FUTEX_WAIT_BITSET_PRIVATE, 2, NULL, 0xffffffff) = ? ERESTARTSYS (To be restarted if SA_RESTART is set)
--- SIGWINCH {si_signo=SIGWINCH, si_code=SI_KERNEL} ---                                                                                                                                         futex(0x7a70e9aa98, FUTEX_WAIT_BITSET_PRIVATE, 2, NULL, 0xffffffff) = ? ERESTARTSYS (To be restarted if SA_RESTART is set)
--- SIGWINCH {si_signo=SIGWINCH, si_code=SI_KERNEL} ---
futex(0x7a70e9aa98, FUTEX_WAIT_BITSET_PRIVATE, 2, NULL, 0xffffffff) = ? ERESTARTSYS (To be restarted if SA_RESTART is set)
--- SIGWINCH {si_signo=SIGWINCH, si_code=SI_KERNEL} ---
futex(0x7a70e9aa98, FUTEX_WAIT_BITSET_PRIVATE, 2, NULL, 0xffffffff) = ? ERESTARTSYS (To be restarted if SA_RESTART is set)
--- SIGWINCH {si_signo=SIGWINCH, si_code=SI_KERNEL} ---
futex(0x7a70e9aa98, FUTEX_WAIT_BITSET_PRIVATE, 2, NULL, 0xffffffff) = ? ERESTARTSYS (To be restarted if SA_RESTART is set)
--- SIGINT {si_signo=SIGINT, si_code=SI_KERNEL} ---                                                                                                                                             +++ killed by SIGINT +++
```

**Comment by kykeon** [comment]
I experience the same. Even something not networking related at all, for instance "torsocks ps" will hang.

Also on aarch64, though not sure that's relevant.

**Comment by xtkoba** [comment]
Presumably already fixed by commit 463f9db605659f8119a06fab7cdfbbb7d5a0dccc (ref. #8464).
